pubtools/_quay/push_docker.py

```diff
@@ -254,8 +254,9 @@ def check_repos_validity(cls, push_items, hub, target_settings):
                     raise
 
             # Check if repo is not deprecated
-            # TODO: check with Comet team if this is a reliable way of checking
-            if "Deprecated" in metadata["release_categories"]:
+            if "Deprecated" in metadata["release_categories"] and target_settings.get(
```

Comment:
Same minor concern as in the related Pub review: no docs + setting name describes the action (check for X) instead of its result (stop push when X).

Feel free to ignore.